include/nbl/builtin/hlsl/glsl_compat/core.hlsl

```diff
@@ -107,12 +110,23 @@ enable_if_t<is_spirv_type_v<Ptr_T>, T> atomicCompSwap(Ptr_T ptr, T value)
 /**
  * GLSL extended math
  */
+
+ // COMMENTING OUT BECAUSE OF https://github.com/microsoft/DirectXShaderCompiler/issues/6751, bring back when fixed!
+ /*
```

Comment:
issue is now resolved, see the issue for new syntax for extended instruction sets